mc_test/callFromMTFExample.py

```diff
@@ -5,16 +5,18 @@
 import os.path, os, binascii
 import random
 
-def generate_params(params_dir, circuit_type, proving_system_type):
+def generate_params(params_dir, circuit_type, proving_system_type, segment_size = 1 << 9, num_constraints = 1 << 10):
```

Comment:
It might be useful to define a constant to store the default segment size and the number of constraints.